supply/spec/uploader_spec.rb

```diff
@@ -158,27 +158,28 @@ def find_obbs
 
       it 'remove lesser than the greatest of any later (i.e. production) track' do
         allow(client).to receive(:track_version_codes) do |track|
-          next [103] if track.eql?('production')
-          next [102] if track.eql?('rollout')
-          next [101] if track.eql?('beta')
+          next [104] if track.eql?('production')
```

Comment:
I don't understand how this code works, but that the number <-> track combination is now different doesn't mean anything?

Reply:
Yeah yeah, so I bumped up the versions so that `production` has highest version and `internal` has lowest version. `internal` needs to be the lowest so I just bumped up the other ones so that I could insert `internal` as the lowest.

Reply:
This tests tried to remove any older versions than the latest production version